dynamixel_controllers/src/dynamixel_controllers/joint_position_controller.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, dxl_io, controller_namespace, port_namespace):
         JointController.__init__(self, dxl_io, controller_namespace, port_namespace)
         
         self.motor_id = rospy.get_param(self.controller_namespace + '/motor/id')
+        self.resolution_divider = rospy.get_param(self.controller_namespace + '/motor/resolution_divider', 1)
+        self.dxl_io.set_resolution_divider(self.motor_id, self.resolution_divider)
```

Comment:
I think we should write this to all other controllers because default dynamixel_io.py considers resolution_divider as 1, which causes error when we use a motor having resolution_divider other than 1 with a controller other than joint_position_controller.
Perhaps dynamixel_io.py should read resolution_divider from motors at some timing.
Also, do you check your code with motors without multi-turn mode (e.g., AX-12)?
I assume this line raises an error.